Dockerfile

```diff
@@ -491,6 +491,11 @@ ADD --chown=buildbot package.json /opt/buildhome/test-env/package.json
 RUN cd /opt/buildhome/test-env && . ~/.nvm/nvm.sh && npm i --legacy-peer-deps &&\
     ln -s /opt/build-bin/run-build-functions.sh /opt/buildhome/test-env/run-build-functions.sh &&\
     ln -s /opt/build-bin/build /opt/buildhome/test-env/run-build.sh
+
 ADD --chown=buildbot tests /opt/buildhome/test-env/tests
 WORKDIR /opt/buildhome/test-env
-CMD . ~/.nvm/nvm.sh && npm test
+
+# Set `bats` as entrypoint
+ENTRYPOINT ["node_modules/.bin/bats"]
+# Set the default flags for `bats`
+CMD ["--recursive", "--timing", "--tap", "tests"]
```

Comment:
Would switching from `CMD` to `ENTRYPOINT + CMD` be a problem when importing inside the buildbot, since it [overrides only `CMD`](https://github.com/netlify/buildbot/blob/1ec6dc389c9f413d6279ea44d1f11466e1123151/Dockerfile#L60)? Wouldn't Docker interpret it then as "overwrite the argument to `bats`"?

Reply:
This will only apply to our [`test` image](https://github.com/netlify/build-image/blob/focal/Dockerfile#L476-L482), which is never used in production (see [here](https://github.com/netlify/build-image/blob/focal/Jenkinsfile#L28) and [here](https://github.com/netlify/build-image/blob/focal/Jenkinsfile#L19), notice the `--target` flag pointing to [build-image](https://github.com/netlify/build-image/blob/focal/Dockerfile#L1). This means that in these contexts the last test layer is actually never built nor used.)

Reply:
Oh, right!